x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/field_select.tsx

```diff
@@ -168,60 +167,76 @@ export function FieldSelect({
     existingFields,
     markAllFieldsCompatible,
   ]);
+  const comboBoxRef = useRef<HTMLInputElement>(null);
+  const labelWidth = (comboBoxRef.current?.clientWidth || 305) - 90; // subtracting paddings
```

Comment:
Probably there are too many "magic numbers" (305, 90, 40...) with only few comments: can we store them in variables with same naming? At least when changing them will understand what are they.